main.py

```diff
@@ -347,3 +347,21 @@ def send_message(message):
         finally:
             # Disconnect from the SMTP server
             smtp.quit()
+
+
+def pubsub(event, context):
+    # Extract the Pub/Sub message data
+    pubsub_message = base64.b64decode(event['data']).decode()
+
+    # Split the message into individual URLs
+    urls = pubsub_message.splitlines()
```

Comment:
Just do this in the loop statement directly. The code would be short enough.

Reply:
Okay